Assembly Tutorial/README.md

```diff
@@ -1,3 +0,0 @@
-This is the Assembly Tutorial. The weather Source produces temperature and wind speed data which triggers the DetectWeatherEmergencyApp. The App uses the isDangerousWeather App Component to determine whether to issue a warning message or evacuation alert to residents. Each alert type will trigger the start of a CollaborationType which will notify the ResidentNotificationSystem Client.
```

Comment:
why was the README removed?
this is what shows up in the Project Import dialog when you select the tutorial...